src/styles/sidebar/components/annotation-omega.scss

```diff
@@ -0,0 +1,20 @@
+// TEMPORARY styles for use during annotation migration
+.annotation-omega {
+  &__controls {
+    display: flex;
+  }
+
+  &__reply-toggle.button {
```

Comment:
Is there a benefit to using the `Button` component vs just a `<button>` element styled appropriately? If we're overriding all of the styling anyway and not using any of its other features then maybe we could do something like:

```diff
diff --git a/src/sidebar/components/annotation-omega.js b/src/sidebar/components/annotation-omega.js
index aba06fa91..93b111323 100644
--- a/src/sidebar/components/annotation-omega.js
+++ b/src/sidebar/components/annotation-omega.js
@@ -110,11 +110,12 @@ function AnnotationOmega({
         {shouldShowLicense && <AnnotationLicense />}
         <div className="annotation-omega__controls">
           {shouldShowThreadToggle && (
-            <Button
-              buttonText={toggleText}
+            <button
               className="annotation-omega__reply-toggle"
               onClick={onToggleThread}
-            />
+            >
+              {toggleText}
+            </button>
           )}
           {shouldShowActions && (
             <div className="annotation-omega-actions">
diff --git a/src/styles/sidebar/components/annotation-omega.scss b/src/styles/sidebar/components/annotation-omega.scss
index 49dcb2c73..8412ee2ce 100644
--- a/src/styles/sidebar/components/annotation-omega.scss
+++ b/src/styles/sidebar/components/annotation-omega.scss
@@ -1,15 +1,17 @@
+@use '../../mixins/buttons' as buttons;
+@use '../../variables' as var;
+
 // TEMPORARY styles for use during annotation migration
 .annotation-omega {
   &__controls {
     display: flex;
   }
 
-  &__reply-toggle.button {
-    padding: 0;
-    background-color: transparent;
-    font-weight: 400;
+  &__reply-toggle {
+    @include buttons.reset-native-btn-styles;
+
+    color: var.$grey-5;
     &:hover {
-      background-color: transparent;
       text-decoration: underline;
     }
   }
```

If we find that we have other instances of buttons that should look the same as this one, then perhaps we can extract a component at that point.

Reply:
I'm gobsmacked by how obvious this notion is, and laughing at myself for not thinking of it. I've been putting so much effort into making every buttonish thing in the app use `Button` that the idea of leaving _some_ as `button` didn't even occur to me as a possible strategy.

I need to think about this some more. Given that I've promised that I'm going to address some other `Button` things next week,  I may need to put this PR/branch on ice (sorry! After so much nag about review!) for a few days.